cms/djangoapps/contentstore/utils.py

```diff
@@ -53,8 +53,11 @@ def initialize_permissions(course_key, user_who_created_course):
     # seed the forums
     seed_permissions_roles(course_key)
 
+    # send legacy enrollment conformation email to course team member
+    studio_request = True
+
     # auto-enroll the course creator in the course so that "View Live" will work.
-    CourseEnrollment.enroll(user_who_created_course, course_key)
+    CourseEnrollment.enroll(user_who_created_course, course_key, studio_request=studio_request)
```

Comment:
Actually, we don't need to pass this extra keyword argument; instead, we can use `settings.ROOT_URLCONF == 'cms.urls'` to determine the system directly inside the `emit_event`.